lib/geography.rb

```diff
@@ -1,9 +1,31 @@
 module Geography
   DEFAULT_COMM_RANGE = 3
-  # assuming that each pixel is a mile, communication range might be approx the distance a human could walk in a mile
+  TERRAIN_TYPES = [
+      "forest",
+      "hills",
+      "mountain",
+      "plains",
+      "marsh",
+      "desert",
+      "ocean",
+  ].freeze
+
+  # assuming that each pixel is a mile, communication range might be approx the distance a human could walk in an hour
```

Comment:
I think we could get pixels to be much higher resolution if you want them. Like 16x16 1k "tiles" would allow us to do 1/16 mi^2 pixels (1/4 mi on each side). Or we could go even higher res if you desire. 

On the other hand, the heightmaps I've seen of britain haven't exceeded 2k pixels on a side. Not sure what resolution we actually have access to without purchasing it from a satellite imaging provider or making friends at Goddard Space Center.